client/components/Cart.jsx

```diff
@@ -1,10 +1,21 @@
 import React, {Component} from 'react'
 import {connect} from 'react-redux'
-import {fetchCart} from '../store.js'
+import {fetchCart, clearCart} from '../store.js'
```

Comment:
You should be getting this from ../reducers, not ../store

Reply:
it may work, but we usually get this from the store.
reducers index basically has a combineReducer and just exports it. The store ultimately stores them. 
Thats why its more accurate to get them from the store, since thats where they live.